### PR TITLE
Add remaining BasicSubscriptions API for adding callbacks for entitle…

### PR DESCRIPTION
**Pull request by stellachui**
…ment and payment responses, as well as expose the auto prompt setup and display function for WP previews.

### CODE REVIEW

**Comment by ChrisAntaki** [comment]
Nice work, very readable

**Review by ChrisAntaki — APPROVED** [review]

File: src/runtime/basic-runtime.js [hunk 1]
```diff
@@ -133,20 +133,39 @@ export class BasicRuntime {
...
+  setOnEntitlementsResponse(callback) {
+    return this.configured_(false).then((runtime) =>
+      runtime.setOnEntitlementsResponse(callback)
+    );
+  }
+
+  /** @override */
+  setOnPaymentResponse(callback) {
+    return this.configured_(false).then((runtime) =>
+      runtime.setOnPaymentResponse(callback)
```
Comment (ChrisAntaki) [inline src/runtime/basic-runtime.js:146]: Could you add a test that covers this?

File: src/runtime/basic-runtime.js [hunk 2]
```diff
@@ -133,20 +133,39 @@ export class BasicRuntime {
...
+  /** @override */
+  setupAndShowAutoPrompt(alwaysShow = false) {
+    return this.configured_(true).then((runtime) =>
+      runtime.setupAndShowAutoPrompt(alwaysShow)
+    );
+  }
+
+  /** @override */
+  dismissSwgUI() {
+    return this.configured_(false).then((runtime) => runtime.dismissSwgUI());
```
Comment (ChrisAntaki) [inline src/runtime/basic-runtime.js:159]: Could you add a test that covers this?

File: src/runtime/basic-runtime.js (hunk 1 above)
Comment (stellachui) [reply src/runtime/basic-runtime.js:146]: Added.

File: src/runtime/basic-runtime.js (hunk 2 above)
Comment (stellachui) [reply src/runtime/basic-runtime.js:159]: Added.

**Pull request merged**